composer.json

```diff
@@ -67,39 +69,20 @@
   "config": {
     "allow-plugins": {
       "dealerdirect/phpcodesniffer-composer-installer": true,
-      "civicrm/composer-downloads-plugin": true,
       "cweagans/composer-patches": true,
       "mcaskill/composer-exclude-files": true,
       "phpstan/extension-installer": true,
-      "composer/installers": true
+      "composer/installers": true,
+      "ergebnis/composer-normalize": true,
+      "bamarni/composer-bin-plugin": true
     },
     "discard-changes": true,
     "platform": {
-      "php": "7.2.24"
+      "php": "7.4"
     },
     "sort-packages": true
   },
   "extra": {
-    "downloads": {
-      "composer-normalize": {
-        "path": "vendor/bin/composer-normalize",
-        "type": "phar",
-        "url": "https://github.com/ergebnis/composer-normalize/releases/download/2.24.0/composer-normalize.phar"
-      },
-      "php-scoper": {
```

Comment:
PHP-Scoper requires PHP 8, so we can't remove this.

Reply:
It required [8.1.](https://github.com/humbug/php-scoper/blob/ee08e096d379100af32275f234951e90a10874a5/composer.json#L20).Using composer to install phpscoper here means we get an older version that is supported. One from September. See https://github.com/humbug/php-scoper/pull/679

Reply:
An _older_ version, I see. The downside is we don't use the latest version and can't upgrade it anymore.

Reply:
But it requires PHP 8.1, something I am not running locally and we only just support. I think running the older version is a benefit here.